package/psyneulinkviewer/node.py

```diff
@@ -0,0 +1,38 @@
+import sys
+import subprocess
+import logging
+import configuration
+from setuptools.command.install import install
+from packaging.version import Version
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+def install_node():
+    logging.info("Installing node")
+    subprocess.run(configuration.conda_forge, shell=True)
+    subprocess.run(configuration.node_installation, shell=True)
+
+def check_node_installation():
+    node_version = subprocess.run(
```

Comment:
same recommendation here, better to use a try except block for these checks, otherwise if the binary is not present it will raise an exception. In this way we can handle that knowing the dependency we are checking we know it's not installed.

Reply:
Done